src/main/mainWindow.js

```diff
@@ -43,7 +40,23 @@ function getValidWindowPosition(state) {
   return state;
 }
 
+function handleScreenResize(screen, browserWindow) {
+  screen.on('display-metrics-changed', () => {
```

Comment:
I thought display removed would lead to a display metrics change. However I'm not sure about that ;)
Adding more handlers does not hurt in this case I guess...